versioned_docs/version-latest/developers/general-information.md

```diff
@@ -46,4 +46,4 @@ The popularity of Solidity is primarily due to its design for Ethereum, the lead
 
 Lastly, Solidity is continually evolving with frequent updates, new features, and improvements that address the unique needs of blockchain development. This responsive development and the thriving community around it further solidify its position as the leading language for smart contract development.
 
-Solidity has a great community of developers and extensive documentation is available on the official [website](https://soliditylang.org/).
\ No newline at end of file
+Solidity has a great community of developers and extensive documentation is available on the official [website](https://soliditylang.org/).
```

Comment:
I can't see what the change is here?

Reply:
This is a result of running the version-update script which brings pre-release and latest to be the same. in the two mentioned cases it was simply newlines to end the doc